src/SimpleXLSXGen.php

```diff
@@ -40,6 +40,8 @@ class SimpleXLSXGen
     protected $keywords;
     protected $category;
     protected $lastModifiedBy;
+    protected bool $prtoectedWorkbook;
```

Comment:
remove type decl, we use PHP 7.0 (prop types added in 7.4)
rename to $isProtectedWorkbook;